packages/perseus-editor/src/editor-page.tsx

```diff
@@ -219,14 +218,6 @@ class EditorPage extends React.Component<Props, State> {
         this.props.onChange(newJson);
     };
 
-    scorePreview(): KEScore | null | undefined {
```

Comment:
`git` history says we [used to use it](https://github.com/khan/webapp/commit/439f898e2019fdb64218f5329675dc3557fafcfd#diff-39dc4b00ccacbe995d660158ef60215369e7e2614fc67b57b4794fec1398769aL81) for demo pages in webapp... that are long gone!